### ISSUE TITLE
Liqiud crytal display library missing

### ISSUE THREAD

**Issue by Matevoo** [swiftforarduino/community#11]

**Comment by carlos4242** [comment]
I think the author is referring to this: https://www.arduino.cc/reference/en/libraries/liquidcrystal/

**Comment by Matevoo** [comment]
> I think the author is referring to this: https://www.arduino.cc/reference/en/libraries/liquidcrystal/

Yes that's what I meant. P.S. I love this project - (SwiftForArduino) :D

**Comment by swiftforarduinoadmin** [comment]
Great! Glad to hear it! :) :)

I don’t know when I’ll have time to get to adding this library, we have to manually import each Arduino classic library when people want them. If you’d like to try the work yourself I’m happy to chat you through it on email? It’s not super complicated, it just takes a bit of time to write all the shims. Contact us on ***@***.*** ***@***.***> if you’d like to try that.

Otherwise, I’ve left this issue open with a “Help Wanted” badge on it so someone can pick it up one day.

Carl

**Comment by Matevoo** [comment]
Oh yea sure I would like to help at some point when I'll gain some further knowledge in swift and C. I can contact you for sure but the email is for some reason hidden in character '*'

**Comment by carlos4242** [comment]
must be a security feature, just go to the swift for arduino website and click on any of the links for sending us an email

**Comment by Matevoo** [comment]
> 

Oh nice I've contacted you on there

**Comment by carlos4242** [comment]
We/I have replied. Sorry it took so long. :)
